### ISSUE TITLE
slow training (learning) of new nnUNetv2

### ISSUE THREAD

**Issue by mehrnia** [MIC-DKFZ/nnUNet#2465]
I was using nnUNetv2 version that I downloaded in April 2024. it was converging in iteration less than 100. 
for the exact same dataset, I'm using nnUNetv2 which I downloaded in August 2024, now it goes to 1000 to converge.
and the dice score is same, what is the reason for this slow performance? I see the learning rate decreases where it may not be.
and how to make it faster. I'm afraid to increase the leaning rate then to sabotage the model.

**Comment by TaWald** [comment]
nnU-Net always used to train 1000 epochs, also in April 2024. 
If you want shorter training time you can write a new trainer class that only trains for 100 epochs if you like.
Try checking out the docu on that because it has all the info you need.

On a side note quicker training schemes generally reduce performance on all tasks (except the very trivial ones), so be aware that you will very likely lose a few dice points.
